core/store/tx_manager.go

```diff
@@ -742,6 +750,7 @@ func (txm *EthTxManager) BumpGasByIncrement(originalGasPrice *big.Int) *big.Int
 	return minimumGasBumpByIncrement
 }
 
+// bumpGas creates a new transaction attempt with an increased gas cost
```

Comment:
It also actually sends the transaction

Reply:
Updated; the term 'create' is used throughout the method.